packages/office-ui-fabric-react/src/common/_semanticSlots.scss

```diff
@@ -49,6 +49,7 @@ $buttonBackgroundHoveredColor: '[theme:buttonBackgroundHovered, default: #eaeaea
 $buttonBackgroundCheckedColor: '[theme:buttonBackgroundChecked, default: #c8c8c8]';
 $buttonBackgroundCheckedHoveredColor: '[theme:buttonBackgroundCheckedHovered, default: #eaeaea]';
 $buttonBackgroundPressedColor: '[theme:buttonBackgroundPressed, default: #eaeaea]';
+$buttonBackgroundDisabledColor: '[theme:buttonBackgroundisabled, default: #ffffff]';
```

Comment:
casing of "disabled" in the string
this also can't be white, use the old styles, not the fluent styles, for now